### ISSUE TITLE
Can't create any report

### ISSUE THREAD

**Issue by chinhodado** [currency-cop/currency-cop#15]
This is the error in the log:

```
[00:59:23] [INFO] [Events] Global notification occurred Unexpected error occurred.
[00:59:23] [ERROR] [Core] Uncaught error: Cannot read property 'id' of undefined - TypeError: Cannot read property 'id' of undefined
    at At.value (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:18:108763)
    at Object.N (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:104711)
    at I (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:100455)
    at Object.executeDispatchesInOrder (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:101516)
    at Y (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:39404)
    at $ (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:39539)
    at Array.forEach (native)
    at u.exports (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:18:72142)
    at Object.processEventQueue (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:40746)
    at N (file:///C:/Users/Chin/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:46:61532)
```

**Comment by MoldyDwarf2** [comment]
Same for me:

	[08:40:18] [INFO] [Core] Loading Currency Cop v1.2.5-beta1
	[08:40:18] [WARN] [Config] File could not be loaded: ENOENT: no such file or directory, open 'C:\Users\<redacted>\AppData\Roaming\currency-cop\User.db'
	[08:40:18] [INFO] [Report] Loading file from: C:\Users\<redacted>\AppData\Roaming\currency-cop\Reports.db
	[08:40:18] [WARN] [Report] File could not be loaded: ENOENT: no such file or directory, open 'C:\Users\<redacted>\AppData\Roaming\currency-cop\Reports.db'
	[08:40:18] [INFO] [Config] Loading file from: C:\Users\<redacted>\AppData\Roaming\currency-cop\User.db
	[08:40:20] [WARN] [API] 404 status response for [https://poe.technology/latest]: "<html>\n<head><title>404 Not Found</title><script src=\"/cdn-cgi/apps/head/kd1PXP1YhrieRZi15hZS76ke7Ro.js\"></script></head>\n<body bgcolor=\"white\">\n<center><h1>404 Not Found</h1></center>\n<hr><center>openresty/1.11.2.4</center>\n</body>\n</html>\n"
	[08:40:33] [WARN] [API] 302 status response for [https://www.pathofexile.com/login]: ""
	[08:40:34] [INFO] [Config] Saving file: C:\Users\<redacted>\AppData\Roaming\currency-cop\User.db
	[08:40:34] [INFO] [Config] Saving file: C:\Users\<redacted>\AppData\Roaming\currency-cop\User.db
	[08:40:37] [INFO] [Events] Global notification occurred Unexpected error occurred.
	[08:40:37] [CRITICAL] [Core] Uncaught error: Cannot read property 'id' of undefined - TypeError: Cannot read property 'id' of undefined
		at jt.value (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:18:106442)
		at Object.w (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:103844)
		at I (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:99585)
		at Object.executeDispatchesInOrder (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:100647)
		at Y (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:40189)
		at Q (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:40324)
		at Array.forEach (<anonymous>)
		at u.exports (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:18:66261)
		at Object.processEventQueue (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:5:41531)
		at w (file:///C:/Users/<redacted>/AppData/Local/Programs/currency-cop/resources/app.asar/dist/bundle.js:46:61958)

Deleting the %APPDATA% folder for currency cop and reinstalling didn't help.

This is on a freshly-installed Win7 VirtualBox VM.
